roles/foreman_installer/tasks/module_prs.yml

```diff
@@ -38,3 +38,10 @@
     loop_var: module
   tags:
     - packages
+
+- include_tasks: module_local.yml
```

Comment:
I think you pushed this file in the AD PR.

Reply:
indeed... fixed :+1: